gen_cookie.py

```diff
@@ -38,6 +38,8 @@ def initBrowser(url):
 browser = initBrowser("https://www.newegg.com")
 app_logger.log_important("Enter cookie file name to save to cookies folder:")
 file_name = input()
+if not file_name:
```

Comment:
will we ever get to this case? how is it possible for file_name to be None, either they enter something or they CTRL-C at that point, no?

Reply:
We would get to this case if the user presses Enter with no content. Ctrl-C would do nothing. This is useful if Enter is entered accidentally or **we just want the default filename instead of having to type it out every time.**

Reply:
Ah, I assumed since file_name would be '' the if statement would not hit. Shows my lack of python knowledge! Good catch.